src/type_level_bool.rs

```diff
@@ -0,0 +1,30 @@
+#[derive(Copy, Clone, Debug)]
+pub struct True;
+#[derive(Copy, Clone, Debug)]
+pub struct False;
```

Comment:
Small nit: I like having `Clone` come before `Copy` in `#[derive]` because 1) `Copy: Clone` and `L < O`.

Reply:
Indeed they're not, I just thought they could come useful eventually. We can always add them back later though